md/installation_guide.md

```diff
@@ -0,0 +1,238 @@
+# Installation guide
+
+Bonita ICI installation guide. It covers evaluation and production mode installation, 
+and detailed available parameters values.  
+
+Installation can be done using *evaluation* or *production* mode. Evaluation mode will start two Docker container 
+and living application. Production mode is a on-premise ICI server deployement.
+
+## Prerequisites
+
+Check all pre-requisites detailed in [Bonitasoft offical documentation](https://documentation.bonitasoft.com)
+
+### Bonita platform
+First of all, you need an up and running **Bonita platform 7.5.4 or greater**.  
+ICI server will connect directly to Bonita database, in a read-only manner.
+
+
+## Evaluation mode
+
+This mode is design to get a fully functional ICI stack including ICI server, ICI storage and living applications 
+for configuration and operation management.
+
+:::warning
+Du to usage of Docker container for ICI server and storage, this mode is not recommended for production platform.
+:::
+
+
+### Linux / OSX
+
+Elasticsearch, even in a Docker container, requires specific configuration of virtual memory. see [elasticsearch documentation](https://www.elastic.co/guide/en/elasticsearch/reference/current/vm-max-map-count.html)
+
+```
+# configure runtime environnement
+sudo sysctl -w vm.max_map_count=262144
+```
+or edit file `/etc/sysctl.conf` file
+
+
+### Interactive installer 
+
+Run the product using `bonita-ici` (`bonita-ici.bat` on Windows platform)script that is located inside the `bin` folder
+
+It will deploy an elasticsearch installation and configure and deploy the ICI application.
+Then it will deploy living application configured to use this ICI installation.
```

Comment:
I don't understand.
Trying something: "Then it will deploy the two Living Applications configured to run on top of this installation".